app/src/pages/BinaryMetricsPage/BinaryMetricsPage.tsx

```diff
@@ -65,6 +65,27 @@ const mapStateToProps = (state: Store): BinaryMetricsDispatchProps => ({
       objKey: aTitle,
     })
   ),
+  confusionMatrix: {
+    totalCount: Math.pow(
+      state.DatasetsStore.selectedDataset?.numberOfRecords ?? 0,
+      2
+    ),
+    falseNegatives: state.MetricsStore.falseNegatives?.data.length ?? undefined,
+    falsePositives: state.MetricsStore.falsePositives?.data.length ?? undefined,
```

Comment:
if state.MetricsStore.falseNegatives is undefinied, then state.MetricsStore.falseNegatives?.data.length returns undefined as well and the suffix " ?? undefinied" is obsolet